nullaway/src/main/java/com/uber/nullaway/handlers/ApacheThriftIsSetHandler.java

```diff
@@ -50,14 +52,16 @@
 
   private static final String TBASE_NAME = "org.apache.thrift.TBase";
 
+  private static final Supplier<Type> TBASE_TYPE_SUPPLIER = Suppliers.typeFromString(TBASE_NAME);
+
   @Nullable private Optional<Type> tbaseType;
 
   @Override
   public void onMatchTopLevelClass(
       NullAway analysis, ClassTree tree, VisitorState state, Symbol.ClassSymbol classSymbol) {
     if (tbaseType == null) {
       tbaseType =
-          Optional.ofNullable(state.getTypeFromString(TBASE_NAME)).map(state.getTypes()::erasure);
+          Optional.ofNullable(TBASE_TYPE_SUPPLIER.get(state)).map(state.getTypes()::erasure);
```

Comment:
Since this code only ever runs once, we could suppress the warning from MemoizeConstantVisitorStateLookups rather than adding the Supplier

Reply:
As long as there isn't a high cost when calling `TBASE_TYPE_SUPPLIER.get(...)` just once, I'd rather use the idiomatic pattern everywhere we can, just in case we ever add a second call for some reason or we copy-paste this somewhere else. So this change is fine by me.